### ISSUE TITLE
Confusing or inaccurate section on try/else in Getting Started Guide

### ISSUE THREAD

**Issue by jlarigakis** [elixir-lang/elixir-lang.github.com#957]
Quoting  [try-catch-and-rescue.markdown](https://github.com/elixir-lang/elixir-lang.github.com/blob/8ed273998f10da1826aa0da2cb44cec7af2ccfbe/getting-started/try-catch-and-rescue.markdown#else): 

```iex
x = 2
try do
  1 / x
rescue
  ArithmeticError ->
    :infinity
else
  y when y < 1 and y > -1 ->
    :small
  _ ->
    :large
end
```

> The result is passed to `else` where it can be pattern matched. Please note that **the `else` block is also executed if an exception has been caught, the result of the `catch/rescue` block will then be passed in**.

If this statement were true, then if we set `x = 0`, the `else` block would receive `:infinity` returned from the `rescue` block, match it as `_` and return `:large`. Changing `x=2` to `x=0` in fact returns `:infinity` in IEX 1.4.2, so it seems that **the else block does not receive the result of a rescue or catch block**.

Please let me know if I've misunderstood.

**Comment by josevalim** [comment]
You are definitely correct: **the else block does not receive the result of a rescue or catch block**. Can you please send a PR?

**Comment by jlarigakis** [comment]
Sure! Fixed it in #961.
